<ui version="4.0" >
 <author>Tomeu Borras Riera</author>
 <class>ArticuloListBase1</class>
 <widget class="QWidget" name="ArticuloListBase1" >
  <property name="enabled" >
   <bool>true</bool>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>690</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Articulos</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../../../bulmages.qrc" >
    <normaloff>:/Images/product-list.svg</normaloff>:/Images/product-list.svg</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>5</number>
   </property>
   <property name="spacing" >
    <number>5</number>
   </property>
   <item row="2" column="0" >
    <widget class="ArticuloList1SubForm1" name="mui_list" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="m_busqueda" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>5</number>
      </property>
      <property name="spacing" >
       <number>5</number>
      </property>
      <item row="0" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>216</width>
          <height>16</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="m_usadoarticulo" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>Solo usados</string>
        </property>
        <property name="shortcut" >
         <string/>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="m_presentablearticulo" >
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text" >
         <string>S&amp;olo presentables</string>
        </property>
        <property name="shortcut" >
         <string>Alt+O</string>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="1" >
       <widget class="BusquedaTipoArticulo" native="1" name="m_tipoarticulo" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item rowspan="3" row="0" column="2" >
       <widget class="BusquedaFamilia" native="1" name="m_familia" >
        <property name="minimumSize" >
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>5</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../bulmages.qrc" >:/Images/product-list.svg</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="m_botonera" >
       <property name="frameShape" >
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" >
        <property name="spacing" >
         <number>5</number>
        </property>
        <property name="margin" >
         <number>5</number>
        </property>
        <item>
         <widget class="QToolButton" name="mui_filtrar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>Filtrar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Filtrar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/edit-find.svg</normaloff>:/Images/edit-find.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_configurar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>Configurar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Configurar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/document-properties.svg</normaloff>:/Images/document-properties.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="mui_actualizar" >
          <property name="minimumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>Actualizar listado</string>
          </property>
          <property name="whatsThis" >
           <string>Actualizar listado</string>
          </property>
          <property name="text" >
           <string/>
          </property>
          <property name="icon" >
           <iconset resource="../../../bulmages.qrc" >
            <normaloff>:/Images/view-refresh.svg</normaloff>:/Images/view-refresh.svg</iconset>
          </property>
          <property name="iconSize" >
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="mui_plugbotones" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Plain</enum>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>161</width>
            <height>34</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Buscar:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="m_filtro" >
          <property name="minimumSize" >
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="focusPolicy" >
           <enum>Qt::ClickFocus</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="5" />
 <customwidgets>
  <customwidget>
   <class>BusquedaTipoArticulo</class>
   <extends>QWidget</extends>
   <header>busquedatipoarticulo.h</header>
  </customwidget>
  <customwidget>
   <class>BusquedaFamilia</class>
   <extends>QWidget</extends>
   <header>busquedafamilia.h</header>
  </customwidget>
  <customwidget>
   <class>ArticuloList1SubForm1</class>
   <extends>QTableWidget</extends>
   <header>articulolisttpv.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mui_list</tabstop>
 </tabstops>
 <resources>
  <include location="../../../bulmages.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>mui_filtrar</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_busqueda</receiver>
   <slot>setShown(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>241</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel" >
     <x>201</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
